cypress/integration/checkin.spec.ts

```diff
@@ -0,0 +1,65 @@
+/// <reference types="cypress" />
+
+import { decrypt } from '../../lib/crypto'
+
+const privateKey = 'KVITtyAWMy0fKTRyqTAm2r05+FkyibXVSnlsI5v0XqQ='
+const publicKey = '5ki/YAX91GQ0ABSyBTsOXBO7tBl6ZJat+OzxnbZCjVM='
+const areaId = '5ac34aab-81f8-4f4d-bc24-97ba8d21eb7b'
+
+context('Checkin', () => {
+  it('has a working checkin', () => {
+    cy.clock(Date.parse('2020-05-11T12:30:00.000Z'), ['Date'])
+
+    cy.intercept('GET', `https://api.rcvr.app/areas/${areaId}`, {
```

Comment:
hmmm...finde es problematisch, dass das auf die live api geht (nicht, dass es versehentlich irgendwelche dinge tut). kann man dem beibringen, dass es heult, wenn andere requests an die api gehen? alternativ kann man vielleicht einfach ein backend via docker starten?

Reply:
Ich füge einfach morgen ein test env hinzu in dem ich den host ändere. Das sollte dann reichen. Ein eigener Server im Docker war ja erstmal nich gewünscht.

Reply:
Done.